rllib/utils/explorations/exploration.py

```diff
@@ -0,0 +1,117 @@
+from ray.rllib.utils.framework import check_framework
+
+
+class Exploration:
+    """
+    An Exploration takes the predicted actions or action values from the agent,
+    and selects the action to actually apply to the environment using some
+    predefined exploration schema.
+    """
+
+    def __init__(self, action_space=None, worker_info=None, framework="tf"):
+        """
+        Args:
+            action_space (Optional[gym.spaces.Space]): The action space in
+                which to explore.
+            worker_info (Optional[dict]): A dict with information on the worker
```

Comment:
It would be cleaner to pass in `worker_index` and `num_workers` as separate variables, unless you have a strong reason to extend it in the future.

Reply:
Yes, makes sense.